RequestService/RequestService.Repo/Repository.cs

```diff
@@ -1082,6 +1098,69 @@ public List<ShiftJob> GetUserShiftJobsByFilter(GetUserShiftJobsByFilterRequest r
                 jobs = jobs.Where(x => request.JobStatusRequest.JobStatuses.Contains((JobStatuses)x.JobStatusId)).ToList();
             };
 
+            if (request.DateFrom.HasValue)
+            {
+                jobs = jobs.Where(x => x.NewRequest.Shift.StartDate >= request.DateFrom.Value).ToList();
```

Comment:
As per comments on a previous PR, I think we need to swap start date and end date on lines 1103 and 1108